.github/workflows/flame_steering_behaviors_pub_publish.yaml

```diff
@@ -18,7 +18,15 @@ jobs:
         uses: actions/checkout@v4
       - name: 🐦 Setup Flutter
         uses: subosito/flutter-action@v2
+      - name: 🪪 Get Id Token
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let pub_token = await core.getIDToken('https://pub.dev')
+            core.exportVariable('PUB_TOKEN', pub_token)
```

Comment:
At the org level I think this is called `PUB_CREDENTIALS`.

Reply:
I think this is not trying to read from the organisation secrets. Instead, it is getting the OIDC credentials using `core.getIDToken` (https://github.com/actions/toolkit/blob/main/packages/core/README.md#oidc-token) and then setting it into a variable named `PUB_TOKEN` to be used later on the Authenticate step.

Reply:
Ah ok.